curity-sgnl-kong/api/server/data/records.json

```diff
@@ -0,0 +1,44 @@
+[
+    {
+        "id": 0,
+        "patient": "Alice",
+        "doctor": "Karen",
+        "region": "USA",
+        "notes": "Lorem ipsum dolor sit amet, consectetur."
```

Comment:
Can we just use bard/gpt to generate some text for each of these?

Reply:
Sure. I do mention that the user can change this data.